pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
Why do we need to increment this ?

Reply:
I think it got auto-updated when I updated my ppm or node version, I required we can push it back to 6.0

Reply:
Let's revert this